index.html

```diff
@@ -11,22 +11,200 @@
 <body>
     <header class="navbar">
         <a class="welcome">Welcome</a>
-        <nav><img class="menu" alt="button" src="Enabled.png" ></nav>
+        <nav><img class="menu" alt="button" src="images/Enabled.png" ></nav>
     </header>
     <section class="main">
```

Comment:
- [ ] Kindly consider following the Figma design. Try using the right height for the background image, please consider also adding some padding to this section so it is as the provided design

|Your design|Figma design|
|-|-|
|![Screenshot_20220609_162815](https://user-images.githubusercontent.com/25227022/172886693-76a02d6f-bfe7-4486-bad9-017b2cbcac3a.png)|![Screenshot_20220609_162918](https://user-images.githubusercontent.com/25227022/172886733-06094917-a579-4cda-a08c-0665d3bf9236.png)|